CHANGES.rst

```diff
@@ -239,6 +239,8 @@ Bug fixes
 
 - ``astropy.io.fits``
 
+  - ``is_image`` property added to GroupsHDU class. [#4742]
```

Comment:
Also this should go under the section for the next bug fix release (I think 1.1.2 or 1.1.3)--@astrofrog can tell you.